packages/travix-ui-kit/components/badge/badge.js

```diff
@@ -103,6 +105,10 @@ Badge.propTypes = {
    * The Badge's position
    */
   position: PropTypes.oneOf(['top', 'right', 'bottom', 'left']),
+  /**
+   * The Badge's size
+   */
+  size: PropTypes.oneOf(['default', 'small']),
```

Comment:
I'm not sure `default` is a good name for a size value. [Button](https://github.com/Travix-International/ui/blob/master/packages/travix-ui-kit/components/button/button.js) and [Price](https://github.com/Travix-International/ui/blob/master/packages/travix-ui-kit/components/price/price.js) components both have a `size` prop with `PropTypes.oneOf(['xs', 's', 'm', 'l', 'xl'])`. Could we have the same here with only `s` and `m`?